src/sentry/static/sentry/app/views/organizationIntegrations/index.jsx

```diff
@@ -169,21 +215,17 @@ class OrganizationIntegrations extends AsyncComponent {
             {reloading && <StyledLoadingIndicator mini />}
           </PanelHeader>
           <PanelBody>
-            {providers}
-            {applications && (
-              <SentryAppInstallations
-                orgId={this.props.params.orgId}
-                installs={appInstalls}
-                applications={applications}
-              />
-            )}
+            {installed}
+            {uninstalled}
           </PanelBody>
         </Panel>
       </React.Fragment>
     );
   }
 }
 
+const IntegrationRow = styled.div``;
```

Comment:
Do we need this component?

Reply:
Functionally, no, but SentryApps and Integrations use two different components for each row. This makes testing the sorting a lot easier (find all `IntegrationRow` items, regardless of which type they are). Long term, I plan to consolidate these two sets of components into one unified set, but that's down the line.

Reply:
I more meant that this is an empty div with no styling. You could just `<div>` for now.

Reply:
They're using the component name as a selector in tests (which I find a bit more pleasant to read)